llvm_jni/build.gradle

```diff
@@ -86,7 +86,7 @@ testBuild.dependsOn(copyNbal, copyJNILLVM, copyDriver, copyDriverOutput, copyLoc
 
 task balCompile(type: Exec) {
         workingDir tmpBuildDir
-        commandLine 'bal', 'build', '--offline' , '--skip-tests'
```

Comment:
These seems to be not in use, I removed just in case. @heshanpadmasiri Please do a cleanup when you get a chance.